web/modules/custom/vehicle_custom/src/Service/VehicleLookupService.php

```diff
@@ -47,12 +47,11 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager, Logge
   public function getNodeTitle($node_id){
```

Comment:
Take a look at [this](https://www.drupal.org/docs/develop/standards/php/php-coding-standards#s-parameter-and-return-type-hinting)

Reply:
As well as this: https://www.drupal.org/docs/develop/standards/php/php-coding-standards#s-functions-and-variables

I know we dont do it in zoocha_reporting, but best to follow when possible!